src/PersonObjects/Sleeve/ui/SleeveElem.tsx

```diff
@@ -31,15 +31,17 @@ function getWorkDescription(sleeve: Sleeve, progress: number): string {
       return "This sleeve is currently set to synchronize with the original consciousness. This causes the Sleeve's synchronization to increase.";
     case SleeveWorkType.BLADEBURNER:
       return (
-        `This sleeve is currently attempting to perform ${work.actionName}.\n\n` +
-        `Progress: ${formatPercent(progress)}`
+        `This sleeve is currently attempting to perform ${work.actionName}.\n\nTasks Completed: ${formatNumber(
+          work.tasksCompleted,
+        )}\n \n` + `Progress: ${formatPercent(progress)}`
       );
     case SleeveWorkType.CRIME: {
       const crime = work.getCrime();
       return (
         `This sleeve is currently attempting ${crime.workName} (Success Rate: ${formatPercent(
           crime.successRate(sleeve),
-        )}).\n\n` + `Progress: ${formatPercent(progress)}`
+        )}).\n\nTasks Completed: ${formatNumber(work.tasksCompleted)} 
```

Comment:
Given that this is an integer, it is probably not useful to include decimal places. There is a `formatInt` function that provides sane defaults for displaying integers (no decimals until suffix is shown, suffix begins at 1000, so the display would show `999`, and then the next value would be `1.000k`)